docs/multi-echo_walkthrough.rst

```diff
@@ -0,0 +1,314 @@
+########################
+What is multi-echo fMRI?
+########################
+
+.. admonition:: TL;DR
+
+    Most echo-planar image (EPI) sequences collect a single brain image following
+    a radio frequency (RF) pulse, at a rate known as the repetition time (TR).
+    This typical approach is known as single-echo fMRI.
+
+    In contrast, multi-echo (ME) fMRI refers to collecting data at multiple echo times,
+    resulting in multiple volumes with varying levels of contrast acquired per RF pulse.
+    Multi-echo fMRI can be used to identify and remove certain types of noise
+    that can't be removed from single-echo data.
+
+To understand what multi-echo fMRI is and why it's useful,
+we will walk through a number of things.
+
+
+*******************
+The physics of fMRI
+*******************
+
+In a typical fMRI sequence,
+the protons in a brain slice are aligned perpendicularly to the main magnetic
+field of the scanner (:math:`B_0`) with an "excitation pulse".
+Those protons start releasing energy from the excitation pulse as they fall back in line with :math:`B_0`,
+and that energy is actively measured at an "echo time" (also known as `TE`_).
+
+The time it takes for 37% of the protons fall back in line with :math:`B_0`
+is known as the transverse relaxation time, or "T2".
+The exact value of T2 varies across voxels in the brain, based on tissue type,
+which is why protocols which are T2-weighted or which quantitatively measure T2 are useful for structural analyses.
+
+fMRI works based on the fact that differences in blood oxygenation
+(indicative of gross differences in neural metabolism ostensibly driven by neural activity)
+also impact observed T2, also known as T2*.
+Thus, changes in the measured signal from an fMRI sequence reflect (at least in part) changes in blood oxygenation.
+
+
+*****************************
+BOLD signal and BOLD contrast
+*****************************
+
+As mentioned above, energy is released after the excitation pulse, and this is our observed fMRI signal.
+The echo time is the point at which that signal is recorded.
+
+Let's take a look at how fMRI signal varies as a function of echo time.
+
+.. image:: https://osf.io/m7aw3/
+   :alt: physics_signal_decay.png
```

Comment:
@emdupre this doesn't seem to be working. Any ideas?

Reply:
I just requested access to that OSF link to see what it includes !

Reply:
I think you should modify to the "render" link, e.g. 
https://mfr.osf.io/render?url=https://osf.io/mx4ku/?direct%26mode=render%26action=download%26mode=render

But updating the url as appropriate !

Reply:
Hm... it seems like it should work, but it doesn't seem to help:

![image](https://user-images.githubusercontent.com/8228902/127582563-96adbe81-9e69-4d5e-af2d-cdc1b94c0a91.png)

Reply:
Here's what's happening and (at least) two suggestions.
During the build process, Sphinx is trying to copy the files into the generated `_build` directory. Because these are embedded images on a web page, this process fails and results in what you're showing above. To resolve this, we could (in no particular order) :

- Update those image tags to iframes, to embed the web pages
- Fetch the relevant files using cURL or the OSF API during the documentation build process so they would exist appropriately in the `_build` directory.
- Switch to another provider (or revert to GitHub) where we can access the files themselves.

Do you have a preference, @tsalo  ?

Reply:
Thanks for digging into this.

>Do you have a preference, @tsalo ?

I'm not sure. I'm leaning toward iframes for right now, but I also really want to try out the Jupyter Book idea, in which case the GIFs would be generated by, and rendered in, the notebooks.